### ISSUE TITLE
Instructor: help page: add questions

### ISSUE THREAD

**Issue by craaaa** [TEAMMATES/teammates#8769]
From https://github.com/TEAMMATES/teammates/issues/8635#issuecomment-378848376 

Add the following questions and answers into `instructorHelp.jsp`:
- When do students get to see responses?
- Can I change the visibility settings after the session has started? 
Yes, but not recommended.
- Can students edit responses after they have submitted?
- Can I set a question as compulsory?
Not at the moment. But can mention in the question description that it is a compulsory question (not enforced).
- Can I extend the deadline?
- Can I extend the deadline to specific students? 
Not at the moment. But can ask the student to view the submission page and send you the responses via email, which you can submit on behalf of the student (give more info on how to do that).
- Can I see when a student tried to access the submission page? 
Not at the moment. If there is such a need, contact us and we'll try to find that information for you.
- How can I resend links to view session results?
- A student says h/she did not receive the link to submit responses. What do I do?
- Is TEAMMATES free? How long is it going to stay free?
- Is data of my courses in TEAMMATES used for research? Are they shared with third parties?
- Is TEAMMATES compliant with international data protection laws?
- Does TEAMMATES cater for daylight saving time adjustments?

~~On hold until #8735 is merged~~

If submitting a PR for this, please also refer #11049, rather than start from scratch.

**Comment by craaaa** [comment]
Additional questions:

- How can I resend links to view session results?
- A student says h/she did not receive the link to submit responses. What do I do?
- Is TEAMMATES free? How long is it going to stay free?
- Is data of my courses in TEAMMATES used for research? Are they shared with third parties?
- Is TEAMMATES compliant with international data protection laws?
- Does TEAMMATES cater for daylight saving time adjustments?

**Comment by Praisey** [comment]
Hi, I will take this issue as #8735 is already merged!

**Comment by studentPavel** [comment]
It is still available?

**Comment by t-cheepeng** [comment]
Hi @studentPavel 
You are free to take up this issue. Do checkout the [developer guide](https://github.com/TEAMMATES/teammates/blob/master/docs/setting-up.md) for setup. You can first suggest some possible answers to the question here before doing up a PR. Thank you.

**Comment by nk183** [comment]
@craaaa  It is still available?

**Comment by Suiiyush** [comment]
Can I work on this issue? I am new to Github it will be my first issue.

**Comment by CreaTorAleXander** [comment]
Is this issue free or is someone working on it?

**Comment by rakeshmissioncode** [comment]
I'd like to start contributing to this issue.
Any hints on which part of the code I should start with, for this issue?

**Comment by Grahamet** [comment]
Hi, I have been working on this, and I was wondering who would be best to talk to about some of the answers to the above 
questions?

**Comment by anshuman-dev** [comment]
Hi. I wish to work on this Issue. Is there anyone working on it already?

**Comment by Grahamet** [comment]
I'm currently working on it and intend to have a pull request either tomorrow or Thursday but they haven't assigned anyone to it.

**Comment by moziliar** [comment]
@Grahamet will assign to you then. Thanks for the update.

**Comment by moziliar** [comment]
Some update about this issue: the `instructorHelp.jsp` content has been moved to `src/web/app/pages-help/instructor-help-page/instructor-help-getting-started/instructor-help-getting-started.component.html`

**Comment by anshuman-dev** [comment]
> I'm currently working on it and intend to have a pull request either tomorrow or Thursday but they haven't assigned anyone to it.

Okay. All the Best! :)

**Comment by Grahamet** [comment]
@moziliar While I have you here I was wondering i was unsure on the answer for these 2 questions and was wondering if you could give some input?

Is data of my courses in TEAMMATES used for research? Are they shared with third parties?
Is TEAMMATES compliant with international data protection laws?

**Comment by moziliar** [comment]
@Grahamet I think I am not qualified to answer those and will divert them to @damithc

**Comment by Grahamet** [comment]
Alright thank you

**Comment by damithc** [comment]
@Grahamet my answers below:

> Is data of my courses in TEAMMATES used for research? Are they shared with third parties?

No. If we decide to change this policy, we'll inform users and will provide an opt out.

> Is TEAMMATES compliant with international data protection laws?

Email us at `teammates@comp.nus.edu.sg` and we'll send you a copy of a document explaining our data protection measures.

**Comment by KishkinJ10** [comment]
can i work on this please i have just started my open source journey and i want to participate in GSoC 2021 ?

**Comment by moziliar** [comment]
@KishkinJ10 thank you for your interest in working on the project. Do note that there is already existing open PR on this issue. Please do look for other issues that might interest you. We would love to offer guidance wherever possible :)

**Comment by A-GHOSH-dev** [comment]
I think this PR is closed now, Can I contribute?

**Comment by t-cheepeng** [comment]
@A-GHOSH-dev Hi, you can go ahead and contribute to this issue

**Comment by damithc** [comment]
> I think this PR is closed now, Can I contribute?

@A-GHOSH-dev if you create a PR for this, also consider starting with the code from #8769 as substantial amount of work has been done in thee already

**Comment by Ln11211** [comment]
hello, is this issue still open for contribution ?, if yes, where can I start ?

**Comment by daongochieu2810** [comment]
@Ln11211 In IntelliJ, you can use Ctrl + Shift + N to search for a file. The issue asks us to add more questions into `instructorHelp.jsp`, so lets search for that file, see how the current questions were added, and add new questions specified in the issue. Rmb to look at #11049 so you dont have to start from scratch

**Comment by NicolasCwy** [comment]
Hi if its ok I would like to contribute to this!

**Comment by jameswalshe** [comment]
Is this issue available?

**Comment by daongochieu2810** [comment]
@jameswalshe seems @lidianycs has not been active on this issue. You can take this issue, and remember to check the description before starting

**Comment by hhdqirui** [comment]
Hi @jameswalshe , the questions are in the other folders, that is `src/web/app/pages-help/instructor-help-page/instructor-help*section`. You can have a check.

**Comment by wkurniawan07** [comment]
Issue has been resolved.
